docs/api.md

```diff
@@ -783,24 +785,54 @@ https://api-accounts.dev.lcip.org/v1/recovery_email/status \
 
 ### Response
 
-Successful requests will produce a "200 OK" response with the account email and details on the verification status in the JSON body object:
+Successful requests will produce a "200 OK" response with the account email and details on the verification status in the JSON body object. A "200" does not mean verified, but instead a tagged union of the status will be returned in JSON:
+
+#### Success
 
 ```json
 {
+  "status": "success",
   "email": "me@example.com",
   "verified": true,
   "sessionVerified": true,
   "emailVerified": true
 }
 ```
 
+#### Pending
+
+```json
+{
+  "status": "pending",
+  "email": "me@example.com"
+}
+```
+
+#### Error
+
+```json
+{
+  "status": "error",
+  "email": "me@example.com",
+  "error": 135
```

Comment:
Just a note that we'll have to be careful here w.r.t. backwards compatibility.  Existing desktop clients will poll this endpoint looking for a "verified" property, and will keep polling until they find that it's "true".  I'm not sure how they'll respond if it's missing entirely, and I don't think there's anything clever we can do to get them to stop polling, apart from returning an error.  We'll have to test the behaviour on all three client platforms once this is in a dev environment.